lib/quartz/scheduler.rb

```diff
@@ -1,71 +1,66 @@
-java_import 'org.quartz.JobKey'
-java_import 'org.quartz.JobBuilder'
-java_import 'org.quartz.TriggerBuilder'
-java_import 'org.quartz.impl.StdSchedulerFactory'
-java_import 'org.quartz.SimpleScheduleBuilder'
-java_import 'org.quartz.CronScheduleBuilder'
-
-java_import 'org.quartz.SchedulerException'
-
-java_import 'java.lang.System'
+require 'log4jruby'
+require 'quartz/jars/log4j-1.2.16'                                                                                                    
 
 module Quartz
   module Scheduler
+    java_import "java.lang.System"
+    include_package "org.apache.log4j"
     def self.included(base)
-      base.class_eval do        
+      base.class_eval do
         include InstanceMethods
         extend ClassMethods
         include Singleton
+
+        System.setProperty("org.quartz.threadPool.class", "org.quartz.simpl.SimpleThreadPool")
+        System.setProperty("org.quartz.threadPool.threadCount", "50")
```

Comment:
Maybe this should be a constant? - I'm assuming 50 is so high we'll never reach it - but it would be nice to not have it hard coded here.